src/perfcollect/perfcollect

```diff
@@ -1625,6 +1665,15 @@ CTRLC_Handler()
 
 EndCollect()
 {
+    # The user must either use "collect stop" or CTRL+C to stop collection.
+    if [ "$operation" == "stop" ]
+    then 
+        # Recover trace info in the previous program.
+        source $collectInfoFile  # TODO: exit and dispose upon missing file
```

Comment:
Agreed that it's important to handle the case where the file isn't there.  Also, it's useful to handle the case where someone calls start two times in a row.

Reply:
I was thinking about this scenario --- currently it seems we don't handle when two collections are on simultaneously? My assumption is we allow this to happen and they just different sessions run in parallel.

Reply:
This could work for LTTng, but I'm not sure it would work for perf.  Either way, I think the key si that we don't want to settings file to get clobbered as a result of multiple runs in different consoles.